packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -122,7 +122,7 @@ export default class QuanticSearchBox extends LightningElement {
           queries: getItemFromLocalStorage(this.localStorageKey) ?? [],
         },
         options: {
-          maxLength: 100,
+          maxLength: 10,
```

Comment:
Did you check if it was still good if we go back to 100?

I don't think it should be 100, but perhaps higher than 10?

Reply:
yes confirmed that and increased to 20.